v20.1/vectorized-execution.md

```diff
@@ -69,6 +89,10 @@ Vectorized execution is supported for the following [data types](data-types.html
 CockroachDB uses the vectorized engine to execute queries on columns with supported data types, even if a column's parent table includes unused columns with unsupported data types.
 {{site.data.alerts.end}}
 
+## Known limitations
+
+Vectorized execution is not as extensively tested as CockroachDB's existing row-oriented execution engine. In addition, [some data types are not supported](#supported-data-types), and some queries are not fully supported.
```

Comment:
I don't think we should say >Vectorized execution is not as extensively tested as CockroachDB's existing row-oriented execution engine

It is tested just as much as the rest of CRDB. Let's delete this line.